### DISCUSSION TITLE
Show hot/shortcut keys

### DISCUSSION THREAD

**Discussion by gertblij** [freeplane/freeplane#343] [Help]
I love the "allocate shortcut keys" (Ctl+command) feature in Freeplane. But now I can't remember them all :-(.
Is there any way I can display all my allocated shortcut keys?

**Comment by gertblij** [comment]
Apologies. I have found the answer but couldn't find a way to delete my original post.

Answer is: Open C:\Users\$user\AppData\Roaming\Freeplane\1.9.x\accelerator.properties

**Comment by EdoFro** [comment]
> Is there any way I can display all my allocated shortcut keys?

You can also see it directly in Freeplane by using menu command  ***Help / Key Reference***
It displays standard and custom hotkeys configurations.

bye,
edo

**Comment by gertblij** [comment]
Thanks, appreciated.

 

Gert
